src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (User it : users) {
```

Comment:
we should give a more meaningful names to our variables
```suggestion
        for (User user : users) {
```